instrumental/drivers/cameras/pco.py

```diff
@@ -25,8 +25,10 @@
 
 log = get_logger(__name__)
 
+
 if PY2:
-    memoryview = buffer  # Needed b/c np.frombuffer is broken on memoryviews in PY2
+    # could not find an equivalent in the future module, so I don't knwow what that buffer is??
+    # memoryview = buffer  # Needed b/c np.frombuffer is broken on memoryviews in PY2
     from time import clock

```

Comment:
I would just leave this as-is. `buffer` is a Python 2 builtin.

Reply:
done